src/mono/sample/iOS-NativeAOT/Program.csproj

```
@@ -0,0 +1,96 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <OutputPath>bin</OutputPath>
+    <IntermediateOutputPath>$(MSBuildThisFileDirectory)/obj/</IntermediateOutputPath>
+    <TargetFramework>net7.0</TargetFramework>
```

this can use NetCoreAppCurrent now:

```suggestion
    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
```

---

Oops, a left-over, corrected.